index.ts

```diff
@@ -8,26 +8,36 @@ export { ActionModule } from './src/app/action/action.module';
 
 // Cards
 export { CardAction } from './src/app/card/card-action/card-action';
-export { CardActionComponent } from './src/app/card/card-action/card-action.component';
+export {
+  CardActionComponent
+} from './src/app/card/card-action/card-action.component';
 export { CardBase } from './src/app/card/card-base';
 export { CardBaseConfig } from './src/app/card/card-base-config';
 export { CardConfig } from './src/app/card/basic-card/card-config';
 export { CardComponent } from './src/app/card/basic-card/card.component';
 export { CardFilter } from './src/app/card/card-filter/card-filter';
-export { CardFilterComponent } from './src/app/card/card-filter/card-filter.component';
-export { CardFilterPosition } from './src/app/card/card-filter/card-filter-position';
+export {
+  CardFilterComponent
+} from './src/app/card/card-filter/card-filter.component';
+export {
+  CardFilterPosition
+} from './src/app/card/card-filter/card-filter-position';
```

Comment:
This seems related to the max-line-length issue. Appears Prettier is breaking line lengths for exports but not imports (imports it will ignore max-line-length). This is an opinionated convention thing it does that I don't think we'd be able to change unless we just ignore max-line-length or set it to a very high number (Prettier would not format this to be multi-line if we do that). 

I can follow up in an upstream issue on this...